js-intro.js

```diff
@@ -84,13 +136,36 @@ var testString = "thisisateststring"
 var toonimals = [ {name: "Itchy", animal: "mouse"}, {name: "Stimpy", animal: "cat"}, {name: "Daffy", animal: "duck"}, {name: "Scratchy", animal: "cat"}, {name: "Ren", animal: "dog"}, {name: "Felix", animal: "cat"}]
 
 // 5a. Write a function that takes in the toonimals array as an argument and returns an array with only the toon objects that are cats. Expected output: [ { name: "Stimpy", animal: "cat" }, { name: "Scratchy", animal: "cat" }, { name: "Felix", animal: "cat" } ]
-
-
+const catFinder = (objects) =>{
+ const nonCats = objects.filter((object) => {
+  return object.animal ! == 'cat'
+ })
+ return nonCats.mao((object) => object.name)
+}
+ console.log("cat finder, catFinder(toonimals"))
 
 
```

Comment:
Really close, just a few tweaks to make this find all the cats:
const catFinder = (objects) =>{
 return objects.filter((object) => {
  return object.animal === 'cat'
 })
}
 console.log("cat finder", catFinder(toonimals))